api/db-cdr/run-bigdata-migrations.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+set -xeuo pipefail
+IFS=$'\n\t'
+
+activity="-PrunList=$1"
+if [ -z ${2+x} ]
+then
+    context=""
+else
+    context="-Pcontexts=$2"
+fi
+
+drop_index_activity="-PrunList=drop_bigdata_indexes"
+add_index_activity="-PrunList=add_bigdata_indexes"
+
+echo "Running big data migrations"
+# Ruby is not installed in our dev container and this script is short, so bash is fine.
+
+CREATE_DB_FILE=/tmp/create_db.sql
+
+function finish {
+  rm -f ${CREATE_DB_FILE}
+}
+trap finish EXIT
+
+cat create_db.sql | envsubst > $CREATE_DB_FILE
+echo "Creating database if it does not exist..."
+mysql -h ${DB_HOST} --port ${DB_PORT} -u root -p${MYSQL_ROOT_PASSWORD} < ${CREATE_DB_FILE}
+
+
+# Download data and import
+REMOTE_DATA_LOC=https://storage.googleapis.com/all-of-us-ehr-dev-peter-speltz
+REMOTE_DATA_BUCKET="gs://all-of-us-ehr-dev-peter-speltz"
+echo "Importing data files from REMOTE_DATA_LOC"
+
+# Add data files to import here
+DATA_FILES=(concept.csv concept_relationship.csv)
+
+# Download data and truncate tables before dropping indexes
+for f in "${DATA_FILES[@]}"
+do
+  local_fpath=/tmp/$f
+  curl -o $local_fpath "$REMOTE_DATA_LOC/$f"
+  #gsutil cp $REMOTE_DATA_BUCKET/$f $f  # todo maybe , gives error now
```

Comment:
Did you run gcloud activate-service-account or gcloud auth login before trying to do this?

We have Ruby commands which copy stuff from GCS without issue:

https://github.com/all-of-us/workbench/blob/master/api/libproject/devstart.rb#L175

Do those work for you?

Reply:
I will check if that was it.

Reply:
couldn't get this to work.  Just going with curl for now. Something about local permission error on my system writing to ~./config/gcloud but I changed the permissions to allow me to write.